spec/features/movies/top40_spec.rb

```diff
@@ -9,29 +9,64 @@
       fill_in :email, with: @user_1.email
       fill_in :password, with: @user_1.password
       click_button 'Login'
-      visit '/movies/top40'
     end
     
     it "I see the top 40 movies in a table" do
-      within('#movies-table') do
-        expect(all('tr').count).to eq(40)
+      VCR.use_cassette('top_0_20') do
+        VCR.use_cassette('top_20_40') do
```

Comment:
It looks as if lines 15 and 16 are repeated quite a few times, I think this could be avoided by including all tests that use it within the same testing block